src/components/SearchForm.vue

```diff
@@ -90,6 +90,10 @@ export default {
   .algolia-autocomplete {
     width: 100%;
   }
+
+  &__input {
+    padding-right: 2.3rem;
+  }
```

Comment:
Can you just change it to `padding-right: 2rem;` ?

Reply:
@Mudassar045 no problem, I just wanted to do the same spacing to the both icon sides